src/config/i18n/ko.json

```diff
@@ -0,0 +1,419 @@
+
+    "secretLockError": "비밀 잠금이 유효하지 않습니다.",
+    "hashLockError": "해시 잠금이 유효하지 않습니다.",
+    "mosaicAddressRestrictionError": "모자이크 주소 제한이 유효하지 않습니다.",
+    "aggregateInnerTransactionsError": "어그리게이트 내부 트랜잭션이 유효하지 않습니다.",
+	"aggregateCosignaturesError": "공동서명자가 유효하지 않습니다.",
+	"accountBalanceTitle": "계좌 잔고",
+	"noAlias": "명칭 없음",
+	"addressBeenCopied": "주소가 클립보드에 복사되었습니다",
+	"failedToCopy": "복사 실패",
+
+    "beneficiaryAddress": "수혜자 주소",
+    "linked": "연결됨",
+    "node": "노드",
+    "vrf": "VRF",
+    "voting": "투표",
+    "nonce": "논스",
+    "action": "액션",
+    "delta": "델타",
+    "harvestedBlock": "하베스트된 블록 (전체)",
+    "transactionGraphic": "트랜잭션 구성도",
+    "resolutionType": "종류",
+    "receiptType": "종류",
+    "stateHash": "상태 해시",
+    "stateHashSubCacheMerkleRoots": "상태 해시 하위 캐시 머클 루트",
+    "blockReceiptsHash": "영수증 해시",
+    "blockTransactionsHash": "트랜잭션 해시",
+    "position": "위치",
+    "hash": "해시",
+    "startEpoch": "최종화 Epoch 시작점",
+    "endEpoch": "최종화 Epoch 시작점",
+    "checkMosaicAddressRestrictionStatus": "모자이크 계좌 제한 확인",
+    "check": "확인",
+    "payloadSize": "크기 (bytes)",
+    "chainHeight": "체인 높이",
+    "finalized": "최종화된 블록",
+    "pending": "최종화 대기 블록",
+    "finalizedHeight": "최종화 높이",
+    "restrictions": "제한 (허용할 경우)",
+    "hashLockTitle": "해시 잠금",
+    "secretLockTitle": "비밀 잠금",
+    "restrictionAdded": "추가됨",
+    "restrictionRemoved": "삭제됨",
+    "statements": "상태",
+    "nodeDetailTitle": "노드 정보보기",
+    "nodeHostDetailTitle": "호스트 정보보기",
+    "nodeLocationMapTitle": "호스트 위치",
+    "nodePublicKey": "공개키",
+    "apiEndpoint": "API Endpoint",
+    "networkGenerationHashSeed": "네트워크가 생성한 시드 해시",
+    "networkIdentifier": "네트워크 구분자",
+    "location": "위치",
+    "ip": "IP",
+    "organization": "조직",
+    "as": "AS",
+    "continent": "대륙",
+    "country": "나라",
+    "region": "지역",
+    "city": "시(구)",
+    "district": "구역",
+    "zip": "Zip",
+    "nodeStatusTitle": "노드 상태",
+    "nodePeerStatusTitle": "피어 노드 상태",
+    "nodeApiStatusTitle": "API 노드 상태",
+    "connectionStatus": "연결",
+    "databaseStatus": "데이터베이스",
+    "apiNodeStatus": "API 노드",
+    "isAvailable": "활성",
+    "lastStatusCheck": "마지막 상태 확인",
+    "nodeChainInfoTitle": "노드 체인 정보",
+    "scoreHigh": "High 스코어",
+    "scoreLow": "Low 스코어",
+    "finalizationEpoch": "최종화 Epoch",
+    "finalizationPoint": "최종화 Point",
+    "finalizedHash": "최종화 해시",
+    "nodeDetailError": "노드 정보 호출에 실패했습니다.",
+    "extendGraphicValue": "정보",
+    "mosaicArtifactId": "아티팩트Id",
+    "namespaceArtifactId": "아티팩트Id",
+    "receiptTitle": "영수증",
+    "created": "생성됨",
+    "ago": "이전",
+    "messageTypeDescriptor_-1": "로우 메시지",
+    "messageTypeDescriptor_0": "일반 메시지",
+    "messageTypeDescriptor_1": "암호화된 메시지",
+    "messageTypeDescriptor_254": "위임 하베스팅 확인 메시지",
+    "votingNode": "투표 노드",
+    "nonVotingNode": "비투표 노드",
+    "votingNodeTooltips": "Sync ahead of common node.",
+    "nonVotingNodeTooltips": "Sync behind of Voting node.",
```

Comment:
Do you like to keep these 2 in English as well?

Reply:
Thank you so much for the detailed review.